buckets/local/repo.go

```diff
@@ -0,0 +1,576 @@
+package local
```

Comment:
The old `Bucket` is now renamed to `Repo`. This thing contains all the blockstore stuff.